lib/public/views/Runs/RunPerPeriod/RunsPerLhcPeriodOverviewPage.js

```diff
@@ -54,42 +60,27 @@ export const RunsPerLhcPeriodOverviewPage = ({ runs: { perLhcPeriodOverviewModel
     const {
         items: remoteRuns,
         detectors: remoteDetectors,
-        lhcPeriodName,
+        lhcPeriod: remoteLhcPeriod,
```

Comment:
These are actually the statistics and not the lhcPeriod, right?
Could you rename accordingly here and at L100, L104? 
This in turn will also help with  L116 where you reference `lhcPeriod.lhcPeriod.name`

Reply:
applied